dist/minimongo-observable/to-observable.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 var observable_cursor_1 = require('./observable-cursor');
+var _ = require('lodash');
+var COLLECTION_EVENTS_DEBOUNCE_TIMEFRAME = 16;
 function toObservable(cursor) {
     var observable = observable_cursor_1.ObservableCursor.create(function (observer) {
```

Comment:
@dotansimha That do you think if save current zone here, i.e. before observable is created, and then use it below to run `observer.next` in. In this way we can choose what zone to use for that (global or angular's one) by running toObserver in that particular zone, i.e. ngZone.run(toObservable(cursor)) or globalZone.run(toObservable(cursor)).